# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/50_mm-2.7-linux.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/config.h.linux	Sun Jun 18 21:59:38 2000
+++ mm2.7/src/config.h	Sun Jun 18 21:59:10 2000
@@ -100,9 +100,11 @@
 #endif
 
 #ifdef SYSV
-#define killpg(a, b) kill(-(a), (b))
-#define bcopy(a, b, c) memcpy(b, a, c)
-#define bzero(a, b) memset(a, 0, b)
+#ifndef linux
+# define killpg(a, b) kill(-(a), (b))
+# define bcopy(a, b, c) memcpy(b, a, c)
+# define bzero(a, b) memset(a, 0, b)
+#endif
 #define bcmp memcmp
 #define index strchr
 #define rindex strrchr
